etc/ci-download.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# This script expects a single argument, which should be an URL pointing to a
+# file for download; it then tries to download that file, into a local file
+# with the same name as the remote file.
+# If a file of the same name exists in ~/.gap-package-cache, then this file
+# is used if it has the same hash
+set -e
+
+PACKAGEDIR=~/.gap-package-cache
+
+name=${*##*/}
```

Comment:
I have no idea what this does.. I can of course look it up, but perhaps you could add a brief comment explaining it?